packaging/greenboot/set-health.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$(readlink -f "$(dirname "${BASH_SOURCE[0]}")")"
+
+if [[ "${SCRIPT_DIR}" == *"green.d"* ]]; then
```

Comment:
This feels tricky. Could we just have 2 scripts so we're explicitly passing the health status?